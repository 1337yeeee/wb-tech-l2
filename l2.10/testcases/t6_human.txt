1K
512
2M
10K
