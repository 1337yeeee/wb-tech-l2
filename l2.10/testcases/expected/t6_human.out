512
1K
10K
2M
